rcgen/src/lib.rs

```diff
@@ -1851,6 +1854,36 @@ mod tests {
 		assert!(extension.value.any);
 	}
 
+	#[test]
+	fn test_with_extended_key_usages_other() {
```

Comment:
I'm surprised this test doesn't need a `cfg` gate on the `x509-parser` feature.

Reply:
It's because `x509-parser` [is also a dev dependency](https://github.com/rustls/rcgen/blob/a3831c9c7714684946276c6b10c981280e6a8297/rcgen/Cargo.toml#L50) besides being under a feature flag

Reply:
Ah! That makes sense and I see existing tests that are already relying on this fact. :+1: